app/src/main/java/org/wikipedia/search/SearchResultsFragment.kt

```diff
@@ -83,7 +84,7 @@ class SearchResultsFragment : Fragment() {
                 }
             }
         }
-
+        PlacesEvent.logAction("search_results_impression", "search_view")
```

Comment:
This will be shown when the Search activity is displayed, regardless of whether there are results or not. Is that desired?

Reply:
I have removed this for now and will ask shay if we really need an impression event for this since its so evolving, and we also have a result choice event. I will make a separate pr if we need to add it back.